README.md

```diff
@@ -27,6 +37,18 @@ This repository contains the Dockerfiles I use for Akeneo and other Symfony deve
 
 - **akeneo/akeneo** and **akeneo/akeneo-behat** have been replaced by **akeneo/akeneo-apache**.
 
+### Where to get help:
+
+The Akeneo Community [forum](https://www.akeneo.com/forums/), the Akeneo Community [Slack](https://akeneopim-ug.herokuapp.com)
```

Comment:
Do not mention the forum, we are privileging GitHub and the Slack user group now.